tests/lib/cli.js

```diff
@@ -171,12 +219,16 @@ describe("cli", function() {
 
     describe("when calling execute more than once", function() {
         it("should not print the results from previous execution", function() {
-            cli.execute("--no-ignore tests/fixtures/missing-semicolon.js");
+            var configPath = getFixturePath("configurations", "semi-error.json");
+            var filePath = getFixturePath("missing-semicolon.js");
+            var passingPath = getFixturePath("missing-semicolon.js");
```

Comment:
Sorry, I've done so much copy-pasting and so little sleeping that my mind is pretty fuzzy at this point.  I'll get there eventually....